### PR TITLE
Use toasts instead of alert for errors

### LINKED ISSUE #159: Use toasts instead of alert for errors
See: https://chakra-ui.com/toast

### PR DESCRIPTION
**Pull request by Yashs911**
Fixes #159

### CODE REVIEW

**Comment by TransmissionsDev** [comment]
Hey @Yashs911, thank you so much for your PR!

However, the `toast` function takes an object with more info about how to render the toast (more info here: https://chakra-ui.com/toast)

Your PR calls `toast(e)` which will not work, please make another commit that calls toast like this:

```
toast({
          title: "Error!",
          description: e.toString(),
          status: "error",
          duration: 9000,
          isClosable: true,
})
```

File: src/components/pages/DepositModal/AmountSelect.tsx [hunk 1]
```diff
@@ -241,7 +243,7 @@ const AmountSelect = React.memo(
           setAreTransactionsRunning(false);
         }
       } catch (e) {
-        alert(e);
+        toast(e);
```
Comment (TransmissionsDev) [inline src/components/pages/DepositModal/AmountSelect.tsx:246]:
```
toast({
          title: "Error!",
          description: e.toString(),
          status: "error",
          duration: 9000,
          isClosable: true,
})
```

File: src/components/pages/DepositModal/AmountSelect.tsx [hunk 2]
```diff
@@ -169,7 +171,7 @@ const AmountSelect = React.memo(
...
```
Comment (TransmissionsDev) [inline src/components/pages/DepositModal/AmountSelect.tsx:174]:
```
toast({
          title: "Error!",
          description: e.toString(),
          status: "error",
          duration: 9000,
          isClosable: true,
})
```

**Review by TransmissionsDev — CHANGES_REQUESTED** [review]
Follow my comments and then I will happily merge this 👍

File: src/components/pages/DepositModal/AmountSelect.tsx (hunk 2 above)
Comment (Yashs911) [reply src/components/pages/DepositModal/AmountSelect.tsx]: Okay

File: src/components/pages/DepositModal/AmountSelect.tsx [hunk 3]
```diff
@@ -242,7 +250,13 @@ const AmountSelect = React.memo(
...
       } catch (e) {
         Honeybadger.notify(e);
-        alert(e);
+        toast({
+          title: "Error!",
+          description: e.toString(),
+          status: "error",
+          duration: 9000,
+          isClosable: true,
+        })
```
Comment (TransmissionsDev) [inline src/components/pages/DepositModal/AmountSelect.tsx:259]: Can you add a semicolon here?

File: src/components/pages/DepositModal/AmountSelect.tsx [hunk 4]
```diff
@@ -169,7 +171,13 @@ const AmountSelect = React.memo(
...
```
Comment (TransmissionsDev) [inline src/components/pages/DepositModal/AmountSelect.tsx:180]: Please add a semicolon here 👍

**Review by TransmissionsDev — CHANGES_REQUESTED** [review]
Just need two semicolons and I will merge this!

**Review by TransmissionsDev — APPROVED** [review]

**Comment by TransmissionsDev** [comment]
Nevermind, no point in being this picky, mering now thank you very much for your help!

**Pull request merged**